src/platform_impl/linux/x11/mod.rs

```diff
@@ -395,9 +392,10 @@ struct DeviceInfo<'a> {
 impl<'a> DeviceInfo<'a> {
     fn get(xconn: &'a XConnection, device: c_int) -> Option<Self> {
         unsafe {
-            let mut count = mem::uninitialized();
-            let info = (xconn.xinput2.XIQueryDevice)(xconn.display, device, &mut count);
+            let mut count = MaybeUninit::uninit();
+            let info = (xconn.xinput2.XIQueryDevice)(xconn.display, device, count.as_mut_ptr());
             xconn.check_errors().ok().and_then(|_| {
+                let count = count.assume_init();
```

Comment:
Is this copying `count`? Can we put the `let count = count.assume_init();` outside the lambda just to stay on the safe side?